src/theme/typography.js

```diff
@@ -3,7 +3,7 @@ const typography = {
     light: 300,
     regular: 400,
     semiBold: 600,
-    extraBold: 800
+    extraBold: 600 // NONE: `extraBold` TOKEN IS DEPRECATED, USE `semiBold` INSTEAD
```

Comment:
@faichuk could you please add a JSDoc's `deprecated` tag? https://jsdoc.app/tags-deprecated.html
e.g.
```javascript
const typography = {
    // ...
    weight: {
        // ...
        semiBold: 600,
        /** @deprecated Use `semiBold` value instead */
        extraBold: 600,
    },
    // ... 
}
```

This should provide helpful deprecation warning when someone attempts to use this constant within the Aurora codebase

Reply:
Done.